src/main/kotlin/app/revanced/patches/youtube/layout/branding/icon/CustomBrandingIconPatch.kt

```diff
@@ -1,42 +1,34 @@
 package app.revanced.patches.youtube.layout.branding.icon
 
 import app.revanced.patcher.data.ResourceContext
-import app.revanced.patcher.patch.PatchException
 import app.revanced.patcher.patch.options.PatchOption.PatchExtensions.booleanPatchOption
 import app.revanced.patcher.patch.options.PatchOption.PatchExtensions.stringPatchOption
 import app.revanced.patches.youtube.utils.compatibility.Constants.COMPATIBLE_PACKAGE
 import app.revanced.patches.youtube.utils.settings.ResourceUtils.updatePatchStatusIcon
 import app.revanced.patches.youtube.utils.settings.SettingsPatch
 import app.revanced.util.ResourceGroup
 import app.revanced.util.Utils.trimIndentMultiline
+import app.revanced.util.copyFile
 import app.revanced.util.copyResources
 import app.revanced.util.copyXmlNode
 import app.revanced.util.patch.BaseResourcePatch
-import java.io.File
-import java.nio.file.Files
+import app.revanced.util.underBarOrThrow
 
-@Suppress("DEPRECATION", "unused")
+@Suppress("unused")
 object CustomBrandingIconPatch : BaseResourcePatch(
-    name = "Custom branding icon YouTube",
+    name = "Custom branding icon for YouTube",
     description = "Changes the YouTube app icon to the icon specified in options.json.",
     dependencies = setOf(SettingsPatch::class),
     compatiblePackages = COMPATIBLE_PACKAGE,
-    use = false,
 ) {
-    private const val DEFAULT_ICON_KEY = "Revancify Blue"
+    private const val DEFAULT_ICON = "revancify_blue"
 
     private val availableIcon = mapOf(
         "AFN Blue" to "afn_blue",
         "AFN Red" to "afn_red",
         "MMT" to "mmt",
-        "MMT Blue" to "mmt_blue",
```

Comment:
help needed here @anddea

Reply:
Don't change the patch itself; since it adds splash animations, just add resources for it.

I don't see the need to change it. In the merge PR before, it changed the 'appicon' option from key to value. Previously, people could use "mmt blue", "MMT Blue", "Mmt blue", "mmt_blue", or any casing they preferred. Now they are forced to use either "MMT Blue" or "mmt_blue". Isn't this a degradation in quality of life?